checkoff

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import subprocess
+import sys
+
+from ocflib.account.search import user_exists
+
+
+# Format of JSON file should be:
+# {
+#     "roster": [<list of OCF usernames of students>],
+#     "users": {
+#         "username1": [<list of labs checked off>],
+#         ...
+#     },
+#     "labs": {
+#         "a1": [<list of usernames checked off],
+#         ...
+#     },
+# }
+
+DATA_PATH = 'checkoff_data.json'
```

Comment:
Maybe `~decal/sp18/checkoff_data.json` or something to that effect? Then you'd probably want a warning that it can't be run on any machine except supernova, tsunami, etc. (anything with NFS).